Taken from Mandrake, fixes signed/unsigned inconsequence
(not sure, if it could cause problems here?)

===== base.c 1.17 vs edited =====
--- 1.17/fs/proc/base.c	Mon Sep 22 06:16:29 2003
+++ edited/fs/procbase.c	Thu Nov 13 17:49:15 2003
@@ -188,7 +188,7 @@
 		atomic_inc(&mm->mm_users);
 	task_unlock(task);
 	if (mm) {
-		int len = mm->arg_end - mm->arg_start;
+		unsigned int len = mm->arg_end - mm->arg_start;
 		if (len > PAGE_SIZE)
 			len = PAGE_SIZE;
 		res = access_process_vm(task, mm->arg_start, buffer, len, 0);
